sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1064,6 +1065,13 @@ public partial class RetryPolicy : Azure.Core.Pipeline.HttpPipelinePolicy
         protected internal virtual bool ShouldRetry(Azure.Core.HttpMessage message, System.Exception? exception) { throw null; }
         protected internal virtual System.Threading.Tasks.ValueTask<bool> ShouldRetryAsync(Azure.Core.HttpMessage message, System.Exception? exception) { throw null; }
     }
+    public partial class RetryPolicyOptions
+    {
+        public RetryPolicyOptions() { }
+        public Azure.Core.DelayStrategy? DelayStrategy { get { throw null; } set { } }
+        public int MaxRetries { get { throw null; } set { } }
+        public Azure.Core.ResponseClassifier? ResponseClassifier { get { throw null; } set { } }
```

Comment:
It would be good to document the precedence rules for response classifiers.  I believe they are already fairly complex across message/pipeline/requestcontext - with this new customization point, it would be good to firmly establish those and make sure assumptions are validated and regression tested in a test matrix.

Reply:
I've documented in this code comments for now. Do you think it would make sense to include in the Configuration Sample that I'm updating - https://github.com/Azure/azure-sdk-for-net/pull/37715/files#diff-72428161e5066dbbc5e348e6031475d84f40c2b1241d6d4d8503e945bf217469?

Reply:
I've also added a note about RequestContext classifier having the highest precedence in the configuration samples.